src/features/user-profile/pages/UserProfilePage.tsx

```diff
@@ -268,6 +289,66 @@ export const UserProfilePage: FC = () => {
         </Col>
 
         <Col>
+          {tab === 'settings' ? (
+            <>
+              <Row>
+                <Col md={6}>
+                  <Card className='mb-4'>
+                    <Card.Body>
+                      <div>
+                        <h5>
+                          <Trans i18nKey='userProfile.generalHeading'>Theme Preference</Trans>
+                        </h5>
+                        <div className=''>
+                          <p className='text-muted'>Select your preferred mode</p>
+                          <Form>
+                            <Form.Check
+                              type='switch'
+                              id='custom-switch'
+                              label='Dark Mode'
+                              onClick={() => toggleTheme()}
+                            />
```

Comment:
You are not setting the value here. So we are still getting the same issue. Toggle to dark mode then refresh the page and the switch starts to the left.

![image](https://user-images.githubusercontent.com/2287977/211660399-da4b5a6c-8149-46ae-892f-cd1223697833.png)

In order to fix this you need to set the value here based on what theme is active.